### PR TITLE
Sprint report 25

### PR DESCRIPTION
**Pull request by bgeuken**

### CODE REVIEW

File: _posts/development/2017-10-09-sprint-report-10.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+With this workflow, as reviewer it was necessary to go back to the old request, look at which changes where requested and then going back to the new request to review again.
+To improve this workflow a little bit we now show also the comments of the superseded request in the new request.
+<img src="https://user-images.githubusercontent.com/3799140/30970526-aca2417c-a465-11e7-8fcf-ef6e2376ccb8.png" />
+</p>
+
+ <h2> Bugfixes </h2>
+ 
+ <p>
+ It was not possible to delete groups with users. This is now fixed. Thanks <a href='https://github.com/lchiquitto'>Ichiquitto</a> for report this bug.
+ </p>
```
Comment (bgeuken) [inline _posts/development/2017-10-09-sprint-report-10.md]: I wasn't totally sure about this one. It sounds like the short form of the following paragraph. Can we drop this line?

**Comment by bgeuken** [comment]
Would be good if someone could proofread the rabbitmq part. I tried to give people an idea how we get there. But I might remember things in a wrong way, eg. how much effort we took to prepare and clean up the event model.

@hennevogel @evanrolfe @mdeniz @DavidKang @ChrisBr

File: _posts/development/2017-10-09-sprint-report-10.md (hunk 1 above)
Comment (DavidKang) [reply _posts/development/2017-10-09-sprint-report-10.md]: @bgeuken Maybe we can extend it a little. @mschnitzer?

**Comment by bgeuken** [comment]
Oh, and I kept the changes I did in separate commmits. So that we can revert / drop them easily.

File: _posts/development/2017-10-09-sprint-report-10.md (hunk 1 above)
Comment (bgeuken) [reply _posts/development/2017-10-09-sprint-report-10.md]: \*g\* I just realized that that's about two independent bug fixes. I guess we could split them via headers.

Comment (bgeuken) [reply _posts/development/2017-10-09-sprint-report-10.md]: @DavidKang @mschnitzer I added sub-headers to split the two issues and added some more description for the delete group issue.

Please check, if the changes are fine for you guys.

File: _posts/development/2017-10-09-sprint-report-10.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+Take a look into PR<a href="https://github.com/openSUSE/open-build-service/pull/3826">#3826</a>.
+</p>
+
+<p>We also made some changes in the view. Before, only a package group with a specific type and pattern type was shown. Now we show the first package group with type 'image'.</p>
+
+<p>In the Kiwi Editor, we also implemented a new feature that autocompletes the name of the packages getting the information the repositories setup in the image.
+Take a look into PR<a href="https://github.com/openSUSE/open-build-service/pull/3884">#3884</a>.
+</p>
+
+<p>The Kiwi Editor documentation was updated with all the new features until now  and how to use it. <a href="http://openbuildservice.org/help/manuals/obs-best-practices/cha.obs.best-practices.webuiusage.html#kiwi_editor_how_to">See the documentation</a>.</p>
```
Comment (ChrisBr) [inline _posts/development/2017-10-09-sprint-report-10.md]: [nitpick] Double whitespace after now

File: _posts/development/2017-10-09-sprint-report-10.md [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+<img src="/images/posts/sprint_25_rabbitmq.png" style="margin: 30px 0 30px 0;" width="500px" />
+
+<p>
+For example, every comment in a package get's send as 'opensuse.obs.package.comment'.
+While comments made in a requests can be identified as 'opensuse.obs.request.comment'.
+</p>
+
+<p>
+If you have your own OBS instance and want to distinguish between events sent from build.opensuse.org and your own instance, you can configure
+prefix of the queue namein the options.yml. Setting the 'amqp_namespace' to 'my.obs_instance' would result in messages sent as 'my.obs_instance.package.comment' and 'my.obs_instance.request.comment'.
```
Comment (ChrisBr) [inline _posts/development/2017-10-09-sprint-report-10.md]: you can configure **a** prefix

``s/namein/name in/``

File: _posts/development/2017-10-09-sprint-report-10.md [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+</p>
+
+<p>
+There are many other options you can configure. All this is documented in our <a href="">admin guide</a>.
```
Comment (ChrisBr) [inline _posts/development/2017-10-09-sprint-report-10.md]: href is missing!

File: _posts/development/2017-10-09-sprint-report-10.md [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+With this workflow, as reviewer it was necessary to go back to the old request, look at which changes where requested and then going back to the new request to review again.
+To improve this workflow a little bit we now show also the comments of the superseded request in the new request.
+<img src="/images/posts/sprint_25_supersed_request_comments.png" width="850px" alt="Supersed request comment" />
+</p>
+
+<h2>Bugfixes</h2>
+ 
+<h3>Fix deletion of groups</h3>
+<p>
+Deleting groups via the API was resulting in an error, as reported in <a href="https://github.com/openSUSE/open-build-service/issues/3393">Issue#3393</a>. This happened, because Rails only removed the Group instance, but not the associated GroupUser. This then caused an ActiveRecord::NotNullViolation exception.This is now fixed. Thanks <a href='https://github.com/lchiquitto'>Ichiquitto</a> for report this bug.
```
Comment (ChrisBr) [inline _posts/development/2017-10-09-sprint-report-10.md]: Space missing before ``This is now fixed.``

File: _posts/development/2017-10-09-sprint-report-10.md [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+
+<p>
+EuRuKo is a Ruby conference that takes place once a year in a European city. This year the EuRuKo Conference was in Budapest, Hungary, and it was the most crowed of all the EuRuKo Conferences, with 690 attendees. As most of you may know, a big part of OBS is written in Ruby, so the team travelled there to hear new things about Ruby and meet people with the same interests.
+</p>
+
+<p>
+ All the talks where great, we specially recommend you to watch the <a href="https://github.com/matz">Yukihiro Matsumoto</a> and <a href='https://github.com/headius'>Charles Nutter</a> keynotes and Ruby 4.0: To Infinity and Beyond talk by <a href='https://github.com/bbatsov'>Bozhidar Batsov</a>. All of them were technical talks about Ruby and we found them really interesting.
+</p>
+<p>
+However, not all of the talks where that technical, but we also enjoyed them. For example, we had a lot of fun with Sebastian Sogamoso post-morten: The overnight failure. The videos are not yet available, but you will be able to watch them soon!
```
Comment (ChrisBr) [inline _posts/development/2017-10-09-sprint-report-10.md]: s/talks where that/talks were that/

**Comment by bgeuken** [comment]
@ChrisBr Updated. I dropped the empy href for now. Our documentation needs to be updated first and there is some issue with generating it.

**Pull request merged**